documentation/start-documenting.rst

```diff
@@ -64,79 +64,109 @@ Building the documentation
+**On Unix platforms** that support :program:`make`
+(including Linux, macOS and BSD),
+you can create a new :mod:`venv` with the required dependencies using::
 
-   cd Doc
    make venv
-   make html
 
-or alternatively ``make -C Doc/ venv html``.  ``htmlview`` can be used
-instead of ``html`` to conveniently open the docs in a browser once the
-build completes.
+Building the docs with :program:`make` will automatically use this environment
+without you having to activate it.
+
+.. _doc-create-venv-windows:
+
+**On Windows**, or if not using :program:`make`,
+`create a new virtual environment <venv-create_>`__ manually.
+Always be sure to `activate this environment <venv-activate_>`__
+before building the documentation.
+
+
+.. _building-using-make:
+.. _using-make-make-bat:
+.. _doc-build-make:
+
+Build using Make / make.bat
+---------------------------
 
-You can also use ``make help`` to see a list of targets supported by
-:command:`make`.  Note that ``make check`` is automatically run when
-you submit a :ref:`pull request <pullrequest>`, so you should make
-sure that it runs without errors.
+A Unix ``Makefile`` is provided, :cpy-file:`Doc/Makefile`,
+along with a :cpy-file:`Doc/make.bat` batch file for Windows
+that attempts to emulate it as closely as practical.
 
-**On Windows**, the :cpy-file:`Doc/make.bat` batchfile tries to emulate
-:command:`make` as closely as possible, but the venv target is not implemented,
-so you will probably want to make sure you are working in a virtual environment
-before proceeding, otherwise all dependencies will be automatically installed
-on your system.
+.. important::
 
-When ready, run the following from the root of your :ref:`repository clone
-<checkout>` to build the output as HTML::
+   The Windows ``make.bat`` batch file lacks a ``make venv`` target,
+   automatically installing any missing dependencies
+   into your current environment.
+   Make sure the environment you `created above <doc-create-venv-windows>`__
+   is `activated <venv-activate_>`__ before running ``make.bat``.
+
+To build the docs as HTML, run::
 
-   cd Doc
    make html
 
-You can also use ``make help`` to see a list of targets supported by
-:cpy-file:`Doc/make.bat`.
+.. tip:: Substitute ``htmlview`` for ``html`` to open the docs in a web browser
+         once the build completes.
+
+To check the docs for common errors with `sphinx-lint`_
+(which is run on all :ref:`pull requests <pullrequest>`), use::
+
+   make check
+
+To list other supported :program:`Make` targets for your platform, run::
+
+   make help
+
+See :cpy-file:`Doc/README.rst` for more information.
+
+
+.. _using-sphinx-build:
+.. _doc-build-sphinx:
+
+Build using Sphinx directly
+---------------------------
+
+Advanced users may want to invoke Sphinx directly,
+to pass specialized options or to handle specific use cases.
 
-See also :cpy-file:`Doc/README.rst` for more information.
+First, ensure the environment you `created above <doc-create-venv-windows>`__
```

Comment:
```suggestion
First, ensure the environment you `created above <doc-create-venv-windows_>`__
```

Reply:
As above, I actually meant this to be a `:ref:` role instead:

```suggestion
   Make sure the environment you :ref:`created above <doc-create-venv-windows>`
```